subprojects/docs/src/docs/release/notes.md

```diff
@@ -1,8 +1,22 @@
-The Gradle team is pleased to announce the release of Gradle 4.1.
+The Gradle team is pleased to announce the first release candidate for Gradle 4.1.
 
-This release now supports running Gradle on the most recent JDK 9 release (b170+). It also optimizes startup speed, positively affecting the execution time of every build.
+First and foremost, we are excited to announce the new [Worker API](userguide/custom_tasks.html#worker_api) for Gradle plugins, facilitating **safe parallel execution**. We encourage plugin authors to [read the new user guide section](userguide/custom_tasks.html#worker_api) as we plan to leverage this API to enable parallel execution by default in a future release. The [Gradle plugin for Android 3.0 alpha](https://developer.android.com/studio/preview/features/new-android-plugin.html) has already adopted this API, so plugin developers will be in good company.
```

Comment:
I would reword this to something like "We encourage plugin authors to [read the new user guide section](userguide/custom_tasks.html#worker_api) and implement tasks using this API as this enables safe parallel execution of tasks."